python/fastexcel/__init__.py

```diff
@@ -12,7 +12,18 @@
 
 import pyarrow as pa
 
-from ._fastexcel import __version__, _ExcelReader, _ExcelSheet
+from ._fastexcel import (
+    ArrowError,
+    CalamineCellError,
+    CalamineError,
```

Comment:
Could we add docstrings? The doc could hence explain in which case those can happen

Reply:
Woops yes, was planning to do it and completely forgot :facepalm: 6c0a38c